docs/guides/latest/advanced-use-case/pause-backup.md

```diff
@@ -0,0 +1,396 @@
+metadata:
+  name: pause-backup
+  namespace: demo
+spec:
+  repository:
+    name: gcs-repo
+  schedule: "*/1 * * * *"
+  target:
+    ref:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: stash-demo
+    volumeMounts:
+    - name: source-data
+      mountPath: /source/data
+    paths:
+    - /source/data
+  retentionPolicy:
+    name: 'keep-last-5'
+    keepLast: 5
+    prune: true
+```
+
+Let's create the `BackupConfiguration` crd we have shown above,
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/advanced-use-case/pause-backup/backupconfiguration.yaml
+backupconfiguration.stash.appscode.com/pause-backup created
+```
+
+**Verify Sidecar:**
+
+If everything goes well, Stash will inject a sidecar container into the `stash-demo` Deployment to take backup of `/source/data` directory. Let’s check that the sidecar has been injected successfully,
+
+```console
+$ kubectl get pod -n demo
+NAME                          READY   STATUS    RESTARTS   AGE
+stash-demo-7489fcb7f5-jctj4   2/2     Running   0          117s
+stash-demo-7489fcb7f5-mfqps   2/2     Running   0          112s
+stash-demo-7489fcb7f5-zp8c5   2/2     Running   0          115s
+```
+
+Look at the pod. It now has 2 containers. If you view the resource definition of this pod, you will see that there is a container named `stash` which is running `run-backup` command.
+
+**Verify CronJob:**
+
+It will also create a `CronJob` with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
+
+Verify that the `CronJob` has been created using the following command,
+
+```console
+$ watch -n 1 kubectl get backupconfiguration -n demo
+Every 3.0s: kubectl get backupconfiguration -n demo                      suaas-appscode: Thu Aug  1 17:08:08 2019
+
+NAMESPACE   NAME           TASK   SCHEDULE      PAUSED   AGE
+demo        pause-backup          */1 * * * *            27s
+```
+
+**Wait for BackupSession Succeeded:**
+
+Wait for the next schedule for backup. Run the following command to watch `BackupSession` crd,
+
+```console
+$ watch -n 1 kubectl get backupssession -n demo
+Every 3.0s: kubectl get backupssession -n demo                      suaas-appscode: Thu Aug  1 17:43:57 2019
+
+NAMESPACE   NAME                      BACKUPCONFIGURATION   PHASE       AGE
+demo        pause-backup-1564659789   pause-backup          Succeeded   49s
+```
+
+We can see from the above output that the backup session has succeeded. This indicates that the volumes of the Deployment have been backed up in the backend successfully.
+
+## Pause Scheduled Backup
+
+Now, we are going to pause the scheduled backup without deleting respective `BackupConfiguration`. In order to do that, we have to set `spec.paused: true` in the respective `BackupConfiguration` crd.
+
+When we set `spec.paused: true`, the following things are going to happen:
+
+- Respective CronJob will not be removed. However, it will skip creating any new BackupSession for next the schedules.
```

Comment:
@hossainemruz , should we suspend the CronJob ?
https://github.com/kubernetes/api/blob/kubernetes-1.14.0/batch/v1beta1/types.go#L110-L113